pages/about.vue

```diff
@@ -1,102 +1,132 @@
 <template>
-  <div class="flex h-full flex-col w-full">
-    <div class="flex flex-col items-center mx-32">
-      <!-- <svg
-        role="img"
-        alt="heart icon"
-        title="heart icon"
-        class="heart-plus-icon w-24 h-24 mr-1 fill-primary-text-inverted"
-      >
-        <use xlink:href="~/assets/images/heart-plus.svg#heart-plus-svg" />
-      </svg>-->
-      <h1 class="mb-12 font-bold text-4xl">{{ $t('about.heading') }}</h1>
-      <p class="mb-12 text-2xl">{{ $t('about.subheading') }}</p>
-      <p class="mb-6 text-md text-primary-text-muted">{{ $t('about.paragraph1') }}</p>
-      <p class="text-md text-primary-text-muted">{{ $t('about.paragraph2') }}</p>
-    </div>
-    <!-- <div id="members" class="flex flex-col items-center mx-32 my-14">
-            <div id="members-header" class="flex">
-                <div class="h-px w-32 border-currentColor/70 border inline-block self-center"></div>
-                <div class="text-primary/80 text-xl font-bold inline-block mx-4 whitespace-nowrap">Our Team</div>
-                <div class="h-px w-32 border-currentColor/70 border inline-block self-center"></div>
+    <div class="flex h-full flex-col w-full">
+        <div class="flex flex-col items-center mx-32">
+            <img
+                src="../assets/images/heart-plus.png"
+                alt="pink heart with a white plus in the middle to symbolize health"
+                class="my-4 h-20 w-30"
+            />
+            <h1 data-testid="about-heading" class="mb-12 font-bold text-4xl">
+                {{ $t("about.heading") }}
+            </h1>
+            <p data-testid="about-subheading" class="mb-12 text-2xl">
+                {{ $t("about.subheading") }}
+            </p>
+            <p
+                data-testid="about-paragraph1"
+                class="mb-6 text-md text-primary-text-muted"
+            >
+                {{ $t("about.paragraph1") }}
+            </p>
+            <p
```

Comment:
this formatting seems a bit strange, is this after running the linter? 
It could be a conflict if you have `prettier` running as well